### PR TITLE
Update installation step to use v1.2.0 version

### PR DESCRIPTION
**Pull request by asifdxtreme**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
A trivial change to use latest v1.2.0 version which checking out the project from git repo

### CODE REVIEW

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ python3 -V
 git clone https://github.com/sodafoundation/installer.git
 cd installer/ansible
 # Checkout the required version. For example, to checkout v1.1.0 follow
```
Comment (joseph-v) [inline content/soda-gettingstarted/installation-using-ansible.md:101]: Please update comment too.

Comment (asifdxtreme) [reply content/soda-gettingstarted/installation-using-ansible.md]: Updated, thanks for the review

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by joseph-v — APPROVED** [review]
LGTM

**Pull request merged**